DuckDuckGo/HomeViewControllerProtocol.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  HomeViewControllerProtocol.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+protocol HomeViewControllerProtocol: UIViewController {
```

Comment:
Let's avoid calling protocols "Protocol" - I know we have some in the code base already, but imo they need refactored.  From the API Design Guidelines:

* Protocols that describe what something is should read as nouns (e.g. Collection).

* Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting).

I think the protocol here is simply `NewTabPage` but open to suggestion.